action/subchain/protocol.go

```diff
@@ -138,16 +147,13 @@ func (p *Protocol) validateStartSubChain(start *action.StartSubChain, ws state.W
 	if account.Balance.Cmp(big.NewInt(0).Add(start.SecurityDeposit(), start.OperationDeposit())) < 0 {
 		return nil, errors.New("sub-chain owner doesn't have enough balance for operation deposit")
 	}
-	if start.StartHeight() < p.chain.TipHeight()+MinStartHeightDelay {
```

Comment:
This was deleted because TipHeight() will cause deadlock on operation blockchain struct. Then, I realize this check is not necessary. If StartHeight is past already, we could just start chainservice for a sub-chain immediately